src/components/header/Header.component.tsx

```diff
@@ -1,2 +1,40 @@
+import cart from '@/assets/icons/cart.svg';
+import logOut from '@/assets/icons/log_Out.svg';
+import logo from '@/assets/icons/logo.svg';
+import moon from '@/assets/icons/moon.svg';
+import sun from '@/assets/icons/sun.svg';
+import userAddIcon from '@/assets/icons/user_Add.svg';
+import Button from '@/UI/button/Button.components.tsx';
+
 import styles from './header.module.css';
-export const HeaderComponent = () => <header className={styles.header}>Header</header>;
+
+const Header = () => (
```

Comment:
![Screenshot 2024-04-28 at 22 09 13](https://github.com/lesiaturuta/fe-react-2024/assets/115080771/60f920a4-2606-4e09-9793-8acc52c994ed)

for mobile version we have burger for header

Reply:
fixed nav